pkg/arvo/app/hark-store.hoon

```diff
@@ -380,17 +383,27 @@
       (~(put by archive-box) index notification(read %.y))
     (give %archive time index)
   ::
+  ::  if we detect cache inconsistencies, wipe and rebuild
   ++  change-read-status
     |=  [time=@da =index:store read=?]
+    ^+  poke-core
     =.  poke-core  (upd-cache read time index)
-    %_  poke-core
-        notifications
-      %^  jub-orm  notifications  time
-      |=  =timebox:store
-      %+  ~(jab by timebox)  index
-      |=  n=notification:store
-      ?>(!=(read read.n) n(read read))
-    ==
+    =/  t=(unit timebox:store)
```

Comment:
Could we call this a... `tib` or a `tmb` or something? And on 394, can we call that a `not`? Superlapidary seems unnecessary. Surely lapidary is terse enough.